### PR TITLE
Feature/book detail page

### LINKED ISSUE #5: Create page to show more information of the book
# :rocket: Feature

The app only show books as a list, and it would be nice if the user could be redirected to a page with more information of the book.
It doesn't need to be a beautiful page, it only needs to work.

### PR DESCRIPTION
**Pull request by gabLaroche**
I also created a component for the Volume card that we see in the search and the favourites page.

### CODE REVIEW

File: src/services/bookApi.ts [hunk 1]
```diff
@@ -50,13 +57,18 @@ export default {
...
 
     const volume = response.data;
+    console.log(volume)
 
     const serializedVolume = {
       id: volume.id,
       title: volume.volumeInfo.title,
       authors: volume.volumeInfo.authors?.join(', '),
       description: volume.volumeInfo.description,
       image: volume.volumeInfo.imageLinks?.thumbnail,
```
Comment (juanbelieni) [inline src/services/bookApi.ts:85]: Not every book has a big picture. Some that I tested came without a cover.

**Review by juanbelieni — CHANGES_REQUESTED** [review]
Bugs:
- Some books doesn't show a cover at detail page.

Improvements:
- When a book is being fetched, it should show a loading spinner.

**Comment by juanbelieni** [comment]
I did the changes that I proposed early. I would like your review before the merge.

File: src/components/Volume/index.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Volume = (props: VolumeProps) => {
       />
       <div className="volume-info">
         <div className="volume-header">
-          <h1><Link to={`books/${volume.id}`}>{volume.title}</Link></h1>
+          <h1><Link to={`books/${volume.id}`} target="__blank">{volume.title}</Link></h1>
```
Comment (gabLaroche) [inline src/components/Volume/index.tsx]: I would add `rel="noreferrer noopener"` to limit potential security breaches

File: src/components/Volume/index.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ const Volume = (props: VolumeProps) => {
       />
       <div className="volume-info">
         <div className="volume-header">
-          <h1><Link to={`books/${volume.id}`} target="__blank">{volume.title}</Link></h1>
+          <h1><Link to={`books/${volume.id}`} target="_blank">{volume.title}</Link></h1>
```
Comment (gabLaroche) [inline src/components/Volume/index.tsx]: I would add `rel="noreferrer noopener"` to limit potential security breaches

Comment (juanbelieni) [reply src/components/Volume/index.tsx]: Yes, right.

**Review by juanbelieni — APPROVED** [review]

**Pull request merged**